packages/rn-tester/js/examples/Text/TextExample.ios.js

```diff
@@ -1264,4 +1264,17 @@ exports.examples = [
       );
     },
   },
+  {
+    title: 'Text components inheriting color from parent',
+    render: function(): React.Node {
+      return (
+        <View style={{marginTop: 10, marginBottom: 10}}>
+          <Text style={{color: 'red'}}>
+            Outer&nbsp;
+            <Text>Inner</Text>
+          </Text>
+        </View>
+      );
+    },
+  },
```

Comment:
_Could_ we add some form of macOS tags here? That might be slightly more difficult 😅